connectors/src/connectors/microsoft/index.ts

```diff
@@ -34,6 +34,7 @@ import {
 } from "@connectors/connectors/microsoft/temporal/activities";
 import {
   launchMicrosoftFullSyncWorkflow,
+  launchMicrosoftGarbageCollectorWorkflow,
```

Comment:
would actually not name it garbage collection; it's a deletion flow
garbage collection would be for resources we did not explicitly notice that they should be deleted and we are catching up
here, the user said: delete this please and we know about it
saying this because in the future we might have an actual garbage collection workflow, that will look for resources whose deletion we missed, see task https://github.com/dust-tt/dust/issues/5751